tools/dynamatic/dynamatic.cpp

```diff
@@ -81,6 +81,7 @@ namespace {
 struct FrontendState {
   std::string cwd;
   std::string dynamaticPath;
+  std::string polygeistPath = "-1";
```

Comment:
I think it's better to have this be the empty string by default (like `dynamaticPath`). It is consistent with `dynamaticPath`, is more meaningful than "-1" for a path, and is not a string one can set with `set-polygeist-path`